rero_mef/version.py

```diff
@@ -30,4 +30,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = '0.2.0'
+__version__ = '0.3.0'
```

Comment:
Due to a lot of change, i will suggest version '0.6.0' or maybe '0.7.0' 🙈

Reply:
0.6.0 it should be